test/webServiceRequestHandlers.test.ts

```diff
@@ -0,0 +1,76 @@
+// TODO: rewrite all the outputprovider handle tests (old ones kept for reference)
```

Comment:
I'd remove this test class if it's no longer relevant. People can go back in Git history if they need to retrieve at all. Unless there's a reason to preserve?